### DISCUSSION TITLE
Adding new handler/rule type - JSON validation by schema

### DISCUSSION THREAD

**Discussion by strauss69** [mockoon/mockoon#728] [Ideas]
@255kb 
Would be nice to validate the request body if new JSON schema rule enabled. Could be pulled from OpenAPI spec file.
And response body too.

**Comment by 255kb** [comment]
I am not sure to get what you ask for. There is already a body rule that is able to check in the request's JSON body for the presence of a [specific property](https://mockoon.com/docs/latest/route-responses/dynamic-rules/#2-property-name-or-path).

**Comment by sebastien-savalle** [comment]
I have the same requirement, mainly for post, put & patch request.
I would like to ensure that the request body is valid, and optionally return a bad request when this is not compliant with a given schema.
I would be nice to integrate AJV for example. https://ajv.js.org/
